.rubocop_todo.yml

```diff
@@ -500,11 +464,6 @@ Style/RegexpLiteral:
 Style/RescueEnsureAlignment:
   Enabled: false
 
-# Offense count: 46
```

Comment:
@domcleal I'd like to know what solution you'd propose to enabling or disabling cops instead of a democratic one. Also, I wished you had voiced your opinion earlier in the foreman-dev thread instead of waiting until I had a PR open. The original email that proposed disabling this cop went out over a week ago.